# Various metadata from DepthAI cameras

std_msgs/Header header

float32 lens_position
int32 exposure_time
int32 sensitivity
int32 category
int32 instance_number
